src/cmd_line/subparsers/file.ts

```diff
@@ -9,7 +9,7 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     }
 
     var scanner = new Scanner(args);
-    let name = scanner.nextWord();
+    var name = scanner.nextWord();
```

Comment:
Now I'm a little bit confused. Previously I was thinking your way of deciding when to use `let` or `var` is

* If it's inside a block (`{ }`, for loop), then we can use `let`
* If it's just in the very top level of the function, then we use `var`

Since JavaScript does hoisting and moves all declaration to the beginning this `var name` here is almost the same as `var name = ""` below in `parseEditFileInNewWindowCommandArgs` (https://github.com/VSCodeVim/Vim/pull/404/files#diff-16bc6b0f3306c3b8bdc5733a584411ffR27). You must have solid reason for this but I'm sorry that I'm not that clear now, do you mind pointing it out?